uw-frame-components/portal/messages/partials/notifications-list-item.html

```diff
@@ -8,7 +8,7 @@
    layout-align="start center"
    layout-xs="column"
    layout-align-xs="center start">
-  <span><md-icon ng-if="notification.priority">priority_high</md-icon> {{ notification.title }}</span>
+  <span><md-icon ng-if="notification.priority == 'high'">priority_high</md-icon> {{ notification.title }}</span>
```

Comment:
How do non-visual users understand that the notification is a priority notification?